nebari/template/stages/07-kubernetes-services/modules/kubernetes/services/jupyterhub/files/jupyterhub/03-profiles.py

```diff
@@ -59,7 +59,11 @@ def base_profile_home_mounts(username):
             "command": ["sh", "-c", command],
```

Comment:
The changes to this file are not technically needed for this PR.  I made the change so that init containers only mount certain subPaths that the user should have access to rather than mounting the whole home and conda pvcs on the initContainers.  This should be slightly more secure.  Also, my motivation for doing this was that the workflow admission controller checks if any of the initContainers or Containers mounts subPaths they shouldn't (or no subPath) and rejects submitted Argo Workflows if so.  

I plan in the future to copy parts of the spec from the user jupyterlab container (including initContainers) for workflows and so the changes to this file make that possible while still passing the validation step of the validating admission controller.